Azure/azure-cli
# [Security] `az security atp cosmosdb`: Add CLI support for ATP settings (Defender) on Cosmos DB

**Related command**
```
az security atp cosmosdb show--resource-group MyResourceGroup --cosmosdb-account MyCosmosDbAccount 
az security atp cosmosdb update --resource-group MyResourceGroup --cosmosdb-account MyCosmosDbAccount --is-enabled 
true
```
**Description**
Adding support for get/set ATP settings (Defender) on Cosmos DB resources.
The REST api supports both Storage accounts and Cosmos DB accounts, but until now, the CLI supported only Storage accounts.
In this PR, we added support for Cosmos DB.

The REST api: 
https://docs.microsoft.com/en-us/rest/api/securitycenter/advanced-threat-protection/get
https://docs.microsoft.com/en-us/rest/api/securitycenter/advanced-threat-protection/create

**Testing Guide**
See related commands section above

---

This checklist is used to make sure that common guidelines for a pull request are followed.
